blocks/procedures.js

```diff
@@ -478,43 +486,61 @@ Blockly.Blocks['procedures_mutatorarg'] = {
     this.setTooltip(Blockly.Msg.PROCEDURES_MUTATORARG_TOOLTIP);
     this.contextMenu = false;
 
+    // Get a handle on the parent workspace.
+    // TODO: Do I need to delete this during dispose?
+    this.outerWs_ = Blockly.Mutator.findParentWs(this.workspace);
```

Comment:
I would definite clear this.outWs_ during `dispose()`.

Reply:
Moved into the functions that use it.  I don't like the idea of overriding the block's dispose function for this.